scripts/docker_build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# This script creates builds according to our custom build policy, which is:
+# - "master" will always point to the most recent build on master
+# - "latest" will refer to the latest tagged release
+# - Each release "foo" will have a tag "foo"
+#
+# This script is a noop if HEAD is not origin/master OR tagged.
+
+set -exuo pipefail
+
+IMAGES="m3dbnode m3coordinator m3query m3nsch"
```

Comment:
Interesting, should we think about centralizing this in a JSON document or something for the images we build for quay?

Reply:
Yeah good idea. I like the minimal dependencies we have so far, but maybe I can this in a JSON config and generate scripts from that or something.